values.yaml

```diff
@@ -329,6 +329,19 @@ concourse:
     ## Possible values: volume-locality | fewest-build-containers | limit-active-tasks | limit-active-containers | limit-active-volumes
     containerPlacementStrategies: []
 
+    ## A second method by which a worker is selected during container placement
+    ## that is only used for get and nested check steps.
+    ## Possible values: volume-locality | random | fewest-build-containers | limit-active-tasks | limit-active-containers | limit-active-volumes
+    noInputContainerPlacementStrategy:
```

Comment:
Is this one necessary? If noInputContainerPlacementStrategies contains only one strategy, isn't it the same as this one?

Reply:
@xtremerui  I thought here just lists all options with default values. Is it true?

Reply:
hmm, i just notice `containerPlacementStrategy` and `containerPlacementStrategy`. We are good here.